src/lib.rs

```diff
@@ -238,6 +249,7 @@ impl BrowserOptions {
 ///     // ...
 /// }
 /// ```
+#[cfg(not(target_os = "ios"))]
```

Comment:
This and other conditionals are not needed if the signature in `ios.rs` is modified appropriately.